oneops-admin/lib/shared/cookbooks/monitor/files/default/check_process_count.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 PROCESS=$1
+
+if [[ `uname` == *CYGWIN* ]]; then
+  out=`powershell "(get-service $PROCESS).Status"`
```

Comment:
Use get-process and output the exact number of processes, not just binary 0/100 output